src/go/cmd/metadata_merge/metadata_merge.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/bazelbuild/bazel-toolchains/src/go/pkg/metadata"
+)
+
+var (
+	outFile = flag.String("outFile", "", "Output merged YAML file to generate.")
+)
+
+// strArgList implements a command line flag that can be specified multiple
+// times to define a list of values.
+type strArgList struct {
+	// Args is the list of command line flags.
+	Args []string
+}
+
+func (l *strArgList) String() string {
+	return fmt.Sprintf("%v", l.Args)
+}
+
+// Set appends the given value for a particular occurance of the flag to the
+// list of flag values.
+func (l *strArgList) Set(value string) error {
+	l.Args = append(l.Args, value)
+	return nil
+}
+
+// metadataYAML stores the metadata generated by security_check & download_pkgs
+// rules as a single YAML object.
+type metadataYAML struct {
+	// Tags is the list of vulnerability tags read from YAML files generated
```

Comment:
dont refer here to 'vulnerability' directly, just refer to tags in yaml files?

Reply:
Done